index.bs

```diff
@@ -3453,6 +3453,14 @@ When this operation is invoked, the [=authenticator=] MUST perform the following
             ::  return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
         </dl>
 
+        Note: The purpose of this [=authorization gesture=] is not to proceed with creating a credential,
+        but for privacy reasons to authorize disclosure of the fact that
+        <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code> is [=bound credential|bound=] to this [=authenticator=].
+        If the user consents, the [=client=] and [=[RP]=] can detect this and guide the user to use a different [=authenticator=].
+        If the user does not consent,
+        the [=authenticator=] does not reveal that <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code> is bound to it,
+        and responds as if the user simply declined consent to create a credential.
+
```

Comment:
index.bs

Reply:
1. If |requireResidentKey| is [TRUE] and the authenticator cannot store a [=client-side-resident public key credential source=]

Reply:
This looks like accidental noise to me. Let me know if it isn't.